src/legacy/ui/public/agg_response/tabify/__tests__/_get_columns.js

```diff
@@ -21,13 +21,15 @@ import expect from '@kbn/expect';
 import ngMock from 'ng_mock';
 import { tabifyGetColumns } from '../_get_columns';
 import { VisProvider } from '../../../vis';
+import { registerVisTypes } from '../../../../../core_plugins/kbn_vislib_vis_types/public/kbn_vislib_vis_types';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
 describe('get columns', function () {
   let Vis;
   let indexPattern;
 
   beforeEach(ngMock.module('kibana'));
   beforeEach(ngMock.inject(function (Private) {
+    registerVisTypes(Private);
```

Comment:
This is _super_ hacky - this provides visTypes for all the tests since its the first test to run.